src/font/lv_font_dejavu_16_persian_hebrew.c

```diff
@@ -5860,7 +5860,7 @@ static LV_ATTRIBUTE_LARGE_CONST const uint8_t glyph_bitmap[] = {
     0xdd, 0x0, 0x8, 0x7c, 0xf7, 0x3f, 0xa4, 0xb,
     0xfc, 0x40, 0x7, 0xfb,
 
-    /* U+FEFF "﻿" */
+    /* U+FEFF "" */
```

Comment:
> comment in font file had BOM signature. Not sure why the comment was even there.

I think it just happens to share the same byte sequence as a UTF-16BE BOM signature but it's a valid unicode character.

BOM signatures can only come at the beginning. Could you update the CI to only check the beginning of the file?

Reply:
I am not all that proficient in bash/sh scripting and I am not sure how I would go about having it check only the first couple of characters in the files. Maybe someone might be able to shed some light on that.

Out of question, what is the reasoning for using UTF-16 BOM as a font character?

Reply:
This should also be fixed now...